_posts/2019-07-09-10-praw-uzytecznego-designu.md

```diff
@@ -0,0 +1,124 @@
+---
+layout:    post
+title:     "10 praw użytecznego designu"
+published: true
+date:      2019-07-09 23:58:00 +0100
+author:    pswierzko
+tags:
+    - ux
+    - ui
+    - design
+---
+
+W latach 80. Dieter Rams zagubiony w chaosie form i kolorów postanowił coś z tym zrobić. Doszedł do wniosku, że dobry design nie może trzymać się sztywnych ram, ale opierając się na pewnych regułach, można stworzyć dobry produkt. Idąc tym tokiem rozumowania opracował 10 reguł dobrego designu, które w branży zdążyły urosnąć do rangi dziesięciu przykazań, a on sam uchodzi dziś za jednego z najlepszych i najbardziej wpływowych projektantów XX wieku.
```

Comment:
"Doszedł do wniosku, że dobry design nie może trzymać się sztywnych ram, ale opierając się na pewnych regułach, można stworzyć dobry produkt." - przearanżowałbym to zdanie. Nie ma takiego wyrażenia jak "trzymać się sztywnych ram". Fragment zdania po "ale" zastąpił bym czymś w rodzaju "lecz pewne reguły stanowią mocne oparcie dla dobrego produktu". W tej chwili druga część zdania brzmi, jakby stała w kompletnej opozycji do pierwszej części, a, jak rozumiem, tak nie jest.

"on sam uchodzi dziś [...]" - zastanowiłbym się, czy nie warto "odświeżyć" tutaj podmiotu, którym jest Rams, a nie design. Zamiast "on sam" można wstawić "ich autor" przykładowo.